src/one_dragon_qt/app/directory_picker.py

```
@@ -205,12 +252,34 @@ def _on_browse_clicked(self):
-                window.selected_directory = self.selected_path
-                window.close()
+        if not self.selected_path:
+            return
+
+        window = self.window()
+        if not isinstance(window, DirectoryPickerWindow):
+            return
+
+        window.selected_directory = self.selected_path
+
+        # 启动解包，切换到进度页
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)